var/spack/repos/builtin/packages/conquest/package.py

```diff
@@ -78,6 +85,19 @@ def edit(self, spec, prefix):
                 "MULT_KERN =.*", f"MULT_KERN = {self.spec.variants['mult_kern'].value}"
             )
 
+    def build(self, spec, prefix):
+        # Starting from 1.3 there's automated logic in the Makefile that picks
+        # from a list of possible files for system/compiler-specific definitions.
+        # This is useful for manual builds, but since the spack will do its own
+        # automation of compiler-specific flags, we will override it.
+        with working_dir("src"):
+            if self.version <= Version("1.2"):
+                make()
+            else:
+                make("SYSTEM = example")
```

Comment:
Can we override just `build_targets` and `build_directory` instead of the entire phase?